changelog/unreleased/enhancement-tus-backend-chunksize

```diff
@@ -0,0 +1,5 @@
+Enhancement: Use tus chunksize from backend
+
+Instead of using the tus chunksize from the web config, we now use the chunksize coming from the backend (if set).
```

Comment:
Can we remove the config chunksize entirely then?

Reply:
if we can have a sane fallback value without config, then yes

Reply:
The current fallback is `infinity`, I'm not sure if this can be seen as "sane" 😅